tests/integration/minibroker_test.go

```diff
@@ -58,10 +71,65 @@ var _ = Describe("classes", func() {
 		assert func(*apiv1beta1.ServiceInstance, *apiv1beta1.ServiceBinding)
 	}{
 		{
-			name:   "mariadb",
-			plan:   "10-3-22",
-			params: map[string]interface{}{},
-			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {},
+			name: "mariadb",
+			plan: "10-3-22",
+			params: map[string]interface{}{
+				"db": map[string]interface{}{
+					"name": "mydb",
+					"user": "admin",
+				},
+			},
+			assert: func(instance *apiv1beta1.ServiceInstance, binding *apiv1beta1.ServiceBinding) {
+				By("rendering and loading the mariadb client template")
+				tmplPath := path.Join(testDir, "resources", "mariadb_client.tmpl.yaml")
+				values := map[string]interface{}{
+					"DatabaseVersion": "10.3",
+					"SecretName":      binding.Spec.SecretName,
+					"Command": []string{
+						"sh", "-c",
+						"mysql" +
+							" --host=${DATABASE_HOST}" +
+							" --port=${DATABASE_PORT}" +
+							" --user=${DATABASE_USER}" +
+							" --password=${DATABASE_PASSWORD}" +
+							" --database=${DATABASE_NAME}" +
+							" --execute='SELECT 1'",
+					},
+				}
+				obj, err := testutil.LoadKubeSpec(tmplPath, values)
+				Expect(err).NotTo(HaveOccurred())
+
+				By("creating the mariadb client resource")
+				ctx := context.Background()
+				pod, err := kubeClient.CoreV1().Pods(namespace).Create(ctx, obj.(*corev1.Pod), metav1.CreateOptions{})
+				Expect(err).NotTo(HaveOccurred())
+				defer func() {
+					err := kubeClient.CoreV1().Pods(namespace).Delete(ctx, pod.Name, metav1.DeleteOptions{})
+					Expect(err).NotTo(HaveOccurred())
+				}()
+
+				By("asserting the mariadb client completed successfully")
+				retry := 0
+				for {
```

Comment:
Is there a reason we're not doing
```suggestion
				for retry := 0; ; retry++ {
```

Reply:
Too much Bash on my head.

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/63/commits/e6bec0390fa0bd4ff65d4db38f75f5c0d9cb73f8.